config/i18n.json

```
@@ -455,15 +454,17 @@
     "CompareLoadout": "Compare Loadout",
     "ConfirmOverwrite": "Are you sure you want to replace the armor in the loadout \"{{name}}\" with this new set of armor?",
     "CreateLoadout": "Save Loadout",
+    "ElementLockedDescription": ". All items will use their existing element.",
+    "ElementNotLockedDescription": ", and will change element as needed to assign mods.",
```

```suggestion
    "ElementNotLockedDescription": "Items will change element as needed to assign mods.",
```
for i18n purposes (clarity for translators, grammar & syntax differences) i would stick with making these both full sentences.

---

Yeah I totally agree. I very nearly made two of each overarching translation so the translators had full context in each case. Making them full sentences is a nice halfway.